airflow/providers/oracle/hooks/oracle.py

```diff
@@ -272,8 +272,14 @@ def insert_rows(
         :param commit_every: the maximum number of rows to insert in one transaction
             Default 1000, Set greater than 0.
             Set 1 to insert each row in each single transaction
-        :param replace: Whether to replace instead of insert
+        :param replace: Does not do anything.
```

Comment:
I would leave the initial intended use-case of the parameter, and just mention that it is not currently implemented, like how you've done with the warning. Leaving it as `Does not do anything` will be confusing for anyone who sees it later on.

Reply:
Makes sense. I've updated it. Please lmk if the current version is fine.